spec/features/groups/groups_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe "Groups", type: :feature do
+  let(:group) { create(:group) }
+
+  it "shows the group name" do
+    sign_in group.owners.first
+    visit group_path(group)
+    expect(page).to have_text("TestGroup")
```

Comment:
Wouldn't it be better to use `group.name` here? (Same thing applies to the tests in `profile_spec.rb`)